### ISSUE TITLE
Bottom line hidden in blender.community on iPad

### ISSUE THREAD

**Issue by jpatrickburns** [armadillica/dillo#182]
Like it says. Scrolling to the bottom of blender.community forums, I cannot access the “load more” button. I can see it when I pull downwards, but letting go of pull results in button ending up below screen. 

I’d include a video, but this form won’t allow for a video that large.

**Comment by venomgfx** [comment]
I think I know what you mean, the scroll area doesn't take into account the automatic-hide of the top/bottom buttons of the browser.

I spent so much time trying to fix this, Safari does one thing and Chrome does another. I ended up hardcoding the scroll area in pixels and still doesn't work.

I would appreciate any help on this if any web developer is reading.

**Comment by jpatrickburns** [comment]
Hmmm…

So when you say, hard coding, do you mean with absolute values? Like pixels?

Have you tried viewport values? Like making the whole content occupy something like 95vh? Or putting the button in a static element stuck to the bottom of the screen? Or give it a relative position, measured from the bottom?

By the way, I’m a big fan of blender today on YouTube. Has been fun pandemic viewing. Trying to visit the Netherlands, but may not actually get to go until the fall because of infection increase in the Netherlands. 

Take care. 

James Burns
More of a dilettante than a web developer

**Comment by jpatrickburns** [comment]
Pablo:

This may be too obvious, but did you try just a vertical height, without doing a calc()? Like height: 98vh? The calc statement seemed so elaborate… I tend to try to simplify when I run into a design problem. 

I tried to hook up my iPad to troubleshoot the css (in Safari), but was getting weird results… not the CSS but references to the original source python stuff. 

Like I said… bijzonder. 

Maybe I’ll give it another try today. 

James Burns
jamesburnsdesign.com

**Comment by venomgfx** [comment]
Thanks for trying! For the next time you've got a moment to spare, maybe follow [this tutorial](https://www.browserstack.com/guide/how-to-debug-on-iphone) on how to enable the Web Inspector (the first step **Method 1: Debug using the Web Inspector feature on iPhones or iPads**)

**Comment by jpatrickburns** [comment]
Yeah… I kinda know the process, but part of the problem for debugging in my setup has to do with me running older OS on my iMac (High Sierra). I have been avoiding upgrading because of all the software that will break (most notably my CS6 software).

So my up-to-date iPad and my older iMac aren’t always talking to each other properly, apparently, and I can’t install later versions of safari, which require later OSes. 

I swear I had it working once, but cannot get my iPad to show up on the development menu. Oh, well. 

James Burns
jamesburnsdesign.com
